quicklogic/qlf_k4n8/tests/features/CMakeLists.txt

```diff
@@ -1,2 +1,3 @@
 #Add feature specific testcases here 
-#add_subdirectory(dct_mac)
+add_subdirectory(bit_file)
+#add_subdirectory(fasm_file)
```

Comment:
Probably this should be uncommented.

Reply:
yes, the comparison was failing here, where as my local its passing. Need to check why. I agree that small change in yosys or vpr, will result in failure of this testcase. Let's run for few days and if these become bottle neck then need to think of alternate approach.